jsonnet/telemeter/rules.libsonnet

```diff
@@ -35,6 +35,18 @@
                 topk by (_id) (1, max by (_id, managed, ebs_account, internal) (label_replace(label_replace((subscription_labels{support=~"Standard|Premium|Layered"} * 0 + 1) or subscription_labels * 0, "internal", "true", "email_domain", "redhat.com|(.*\\.|^)ibm.com"), "managed", "", "managed", "false")) + on(_id) group_left(version) (topk by (_id) (1, 0*cluster_version{type="current"})))
               |||,
             },
+            {
+              record: 'id:alerts:count',
+              expr: |||
+                count by (_id) (alerts{alertstate='firing', severity='critical'})
+              |||,
+            },
+            {
+              record: 'id:cluster_operator_conditions:count',
```

Comment:
Same as above, possibly may want to change the recording rule name here to clarify that these are only `Failing` operators. See https://github.com/openshift/telemeter/blob/d7dcf4c7a2ba7949a06c36dc12e8e7591d29b77f/jsonnet/telemeter/rules.libsonnet#L15-L18 for an example where recorded metric name provides a hint on what the label selectors used were.